src/Sulu/Component/Rest/Tests/Unit/ListBuilder/ListRestHelperTest.php

```diff
@@ -98,7 +98,8 @@ public static function dataFieldsProvider()
      */
     public function testGetFields($request)
     {
-        $helper = new ListRestHelper($request, $this->em);
+        $this->requestStack->method('getCurrentRequest')->willReturn($request);
```

Comment:
Yeah, almost everywhere. When the PHPUnit mocks are seen this can be considered legacy.